cv_next/app/feed/components/filterPanel.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react"
+import { SearchBox } from "../../../components/ui/filters/searchbar"
+import { DropdownInput } from "../../../components/ui/filters/valueSelect"
+import Categories from "@/types/models/categories"
+import { ModeSwitch } from "@/components/ui/filters/modeSwitch"
+
+export interface filterObj {
+    searchValue: string
+    categoryId: number | null
+}
+
+export const MODES:[string, string] = ["review", "published"]
```

Comment:
Not the best way to define a constant. You'd either want an enum here, or a simple:
`export const MODES = {'Review': 0, 'Published': 1};`
enum is probably the way to go:
`export enum MODES {'Review', 'Published'};`

Reply:
Also this might be better to be in Definitions because it'll probably be used elsewhere as well

Reply:
This will be used in the next version as in this one we decided not to include CV versions.
I've deleted this and the ModeSwitch references but the component itself is still there for the future.